javascript/chronometer.js

```diff
@@ -1,26 +1,33 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval((currentTime) => this.currentTime++, 1000);
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime - Math.floor(this.currentTime / 60) * 60;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(time) {
+    if (time < 10) {
+      time.toString();
+      return `0${time}`;
+    }
   }
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.startClick);
```

Comment:
el método clearInterval() toma como argumento un `intervalId`, el identificador de la acción reiterativa (que se inició mediante una llamada a setInterval en el método startClick) y que se desea cancelar. Entonces lo correcto sería:
`clearInterval(this.intervalId);`